### PR TITLE
DTSPO-6680 TEST: VH Fix overlays and values

### PR DESCRIPTION
**Pull request by mokainos**

### CODE REVIEW

File: k8s/environments/test/common-overlay/flux-patch.yaml [hunk 1]
```diff
@@ -58,3 +58,52 @@ spec:
...
+  name: vh-user-api
+  namespace: vh
+  annotations:
+    fluxcd.io/automated: "true"
+    filter.fluxcd.io/java: glob:prod-*
+spec:
+  values:
+    java:
+      image: sdshmctspublic.azurecr.io/vh/user-api:20210210.7
+    global:
```
Comment (timja) [inline k8s/environments/test/common-overlay/flux-patch.yaml:77]: the global should be included by the environment helm release which should already be working, and if not this isn't the place to add it

**Review by timja — CHANGES_REQUESTED** [review]
removal global and fix lint warning

File: k8s/environments/test/common-overlay/flux-patch.yaml (hunk 1 above)
Comment (mokainos) [reply k8s/environments/test/common-overlay/flux-patch.yaml:77]: https://github.com/hmcts/vh-user-api/tree/feature/VIH-7155_AksSecretFileProvider/charts/vh-user-api
Not included in the repo the helmchart is pointing to.

Comment (timja) [reply k8s/environments/test/common-overlay/flux-patch.yaml:77]: It's this file:
https://github.com/hmcts/sds-flux-config/blob/master/k8s/environments/test/test-helmrelease.yaml

**Review by timja — CHANGES_REQUESTED** [review]
nothing has changed since my last review see comments

**Review by jordanhoey — APPROVED** [review]

**Review by timja — APPROVED** [review]

**Pull request merged**